cmd/snap/cmd_snap_op.go

```diff
@@ -269,7 +270,7 @@ func listRefresh() error {
 		return err
 	}
 	if len(snaps) == 0 {
-		return fmt.Errorf(i18n.G("all snaps are fresh"))
+		return fmt.Errorf(i18n.G("All snaps up-to-date"))
```

Comment:
I think this should also do fmt.Fprintln, to be consistent with listSnaps in cmd_list.go as you can see there